src/topbar/components/KTopbarUI.vue

```diff
@@ -19,8 +19,49 @@
           v-else
           show-loading
         />
-      </RouterLink>
-    </div>
+      </div>
+      <QMenu>
+        <QList>
+          <QItem
+            v-for="group in myGroups"
+            :key="group.id"
+            v-close-popup
+            :to="{ name: 'group', params: { groupId: group.id } }"
+            clickable
+          >
+            <QItemSection
+              avatar
+              class="groups"
+            >
+              <QIcon
+                v-if="!group.hasPhoto"
+                name="fas fa-users"
+                class="groups"
+              />
+              <img
```

Comment:
To avoid errors like `TypeError: Cannot read property 'thumbnail' of undefined` because of `group.photoUrls.thumbnail`, don't try to show the photo if there is none.